docs-site/content/docs/features/authentication.md

```diff
@@ -14,15 +14,17 @@ toc = true
 top = false
 +++
 
+## User Password Authentication
+
 `Loco` simplifies the user authentication process, allowing you to set up a new website quickly. This feature not only saves time but also provides the flexibility to focus on crafting the core logic of your application.
 
-## Authentication Configuration
+### Authentication Configuration
 
 The `auth` feature comes as a default with the library. If desired, you can turn it off and handle authentication manually.
 
-## Getting Started with a Saas App
+### Getting Started with a Saas App
 
-Create your app using the [loco-cli](/docs/getting-started/tour.md) and select the `Saas app` option.
+Create your app using the [loco-cli](/docs/getting-started/tour) and select the `Saas app` option.
```

Comment:
URL is shows 404. Fix it.

Reply:
Does this link work for you when you run the app?
i think you should replace it to 
[loco-cli](@/docs/getting-started/tour.md)

Reply:
Does this link work for you when you run the app?
i think you should replace it to 
[loco-cli](@/docs/getting-started/tour.md)

Reply:
@kaplanelad 
Yes works when run on dev with `zola serve`
I attach SS of dev and prod below.

The reason of it because now `docs-site/content/docs/getting-started/tour/index.md` so there is no `tour.md` anymore.

## Fixed
https://github.com/loco-rs/loco/assets/45615348/7c439d24-112d-4b73-b49d-ce3add91d01f

And I check prod, it NOT works.

https://github.com/loco-rs/loco/assets/45615348/893e7651-3c57-4313-adc5-d00c9179efdc